cmd/notation/cert/generateTest.go

```diff
@@ -83,7 +83,7 @@ func generateTestCert(opts *certGenerateTestOpts) error {
 	fmt.Println("generated certificate expiring on", rsaCertTuple.Cert.NotAfter.Format(time.RFC3339))
 
 	// write private key
-	keyPath, certPath := dir.Path.Localkey(name)
+	keyPath, certPath := dir.LocalKeyPath(name)
```

Comment:
dir.LocalKeyPath is the relative directory. 

To get the abstract directory:
relativeKeyPath, relativeCertPath := dir.LocalKeyPath(name)
keyPath, err := dir.ConfigFS().SysPath(relativeKeyPath)
certPath, err := dir.ConfigFS().SysPath(relativeCertPath)